src/common/hooks/usePaidAccount.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+
+import { ManualAccountType } from '../enums'
+
+export const usePaidAccount = () => {
+  const [isPaid, setIsPaid] = useState(false)
+  const [deleteCookie, setDeleteCookie] = useState(false)
```

Comment:
лучше через `is` наименование тут начать
```suggestion
  const [isDeleteCookie, setIsDeleteCookie] = useState(false)
```